ipaserver/session.py

```diff
@@ -21,6 +21,7 @@
 import os
 import re
 import time
+import io
```

Comment:
Is there any particular reason you are using the io module here? In Python 3 ```io.open is open```. In Python 2 the io module's main use is to auto-encode UTF-8 text. It's not wrong, I'm just curious.